projects/plugins/boost/app/modules/optimizations/minify/class-js.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Optimizations\Minify;
+
+use Automattic\Jetpack_Boost\Contracts\Pluggable;
+
+class Js extends Minify implements Pluggable {
+
+	public static $default_excludes = array( 'jquery', 'jquery-core', 'underscore', 'backbone' );
+
+	public static function get_slug() {
+		return 'minify_js';
+	}
+
+	public function init_concatenate() {
```

Comment:
Both `Js` and `Css` classes do the same thing here - they check for admin, initialize the filesystem. This points to a design patterns smell (in combination with the other feedback about the empty method)

Instead of inheritance, I'd recommend flipping this inverting the dependencies here or maybe getting rid of the Minify class entirely. After all, Js and Css are modules that subscribe to `Pluggable` and they can be turned off and on individually, and the handling of that is already defined by `Pluggable`

Reply:
The initialization has been reworked in 286067045c360986184d090cb638bc9956fd7bcf.